src/shared/common/components/summary/actions/action-summary-interests.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const renderRow = ({ label, value }) => {
+    return (
+        <tr key={label} className='govuk-table__row'>
+            <th className='govuk-table__header padding-left--small govuk-!-width-one-third'>{label}</th>
+            <td className='govuk-table__cell'>{value}</td>
+        </tr>
+    );
+};
+
+renderRow.propTypes = {
+    label: PropTypes.string.isRequired,
+    value: PropTypes.string.isRequired
+};
+
+const ActionSummaryInterests = (props) => {
+
+    const interests = [ ...props.props.items ];
+
+    if (interests.length < 1) {
+        return false;
+    }
```

Comment:
I've changed this to pass in the arrays to both appeals and expected interests and added items to the propTypes. 

I'm happy to change it from false, but I don't think we want to display a message if there are no values. I would expect that if there are no interests then it would not appear in the summary tab.

Reply:
Makes sense - I think when I commented this I'd not considered fully it was the summary tab